JoinColony/colonyDapp
# Add WhitelistAddresses component

Add WhitelistAddresses component (UI only) to whitelist extension flow

- Added `WhitelistAddresses` component to `Whitelist`
- Added `colonyAddress` prop to `Whitelist`
- Added `WhitelistMembersListExtraContent` component to display remove button and user date
-  Added new appearance prop `hoverColor` to `ListGroup`
- Added `showReputation` prop to `MembersList` in order to control when to show user reputation or not
- Added `listGroupAppearance` prop to `MembersList` in order to control the `ListGroup` style inside when using `MembersList`


### How to test
1. Remove `extensionEnabled &&` in order to see the content of Whitelist without enabling the extension
2. Inside `Whitelist`, manually add users to the `const users = [ ]` with an `AnyUser` structure, so something like:
`{ id: ...some wallet address, profile: { walletAddress: ...some wallet address } }`

![FireShot Capture 378 - Colony - localhost](https://user-images.githubusercontent.com/18473896/124815787-0fbdde80-df3e-11eb-8dca-b0c5909a1573.png)

Resolves DEV-444
